packages/github-auth/src/githubAuth.ts

```diff
@@ -0,0 +1,38 @@
+import type { MiddlewareHandler } from 'hono'
+
+import { AuthFlow } from './authFlow'
+import type { GithubScope } from './types'
+
+export function githubAuth(options: {
```

Comment:
Every provider should implement `state` if it's available

Reply:
I forgot it there, my bad.